modules/end-user-guide/partials/ref_devfile-reference.adoc

```diff
@@ -818,7 +818,7 @@ components:
       app: postgres
 ----
 
-Additionally, it is also possible to modify the entrypoints (command and arguments) of the containers present in the resource list. For details of the advanced use case, see xref:end-user-guide:configuring-the-workspace-and-adding-tooling#defining specific-container-images[].
+Additionally, it is also possible to modify the entrypoints (command and arguments) of the containers present in the resource list. For details of the advanced use case, see xref:end-user-guide:creating-and-configuring-a-new-workspace#defining specific-container-images[].
```

Comment:
Even better:

```suggestion
Additionally, it is also possible to modify the entrypoints (command and arguments) of the containers present in the resource list. For details of the advanced use case, see xref:creating-and-configuring-a-new-workspace.adoc#defining specific-container-images[Defining specifc container images].
```
* The reference is in the same guide: the guide name is not necessary
* The reference is a deep link, we need to populate the title   See: https://docs.antora.org/antora/2.3/page/module-and-page-xrefs/#default-link-text

> When the target includes a fragment (aka a deep link) like xref:page.adoc#fragment[…​], the link text (…​) isn’t automatically populated. You’ll need to specify the xref’s link text in this case. 

NB: I am very afraid we have still a lot of broken deep links.